pkgs/development/python-modules/icalendar/default.nix

```
@@ -10,15 +10,15 @@
 }:
 
 buildPythonPackage rec {
-  version = "5.0.10";
+  version = "5.0.12";
   pname = "icalendar";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `build-system`